__tests__/app.test.js

```diff
@@ -283,6 +283,16 @@ describe("PATCH /api/reviews/review_id", () => {
         });
       });
   });
+
+  test("404: responds with correct error status when invalid review_id is used", () => {

```

Comment:
please try to avoid adding fixes to previous tickets to new branches it can be both confusing to read but also it muddies your git history